tests/HUBActionRegistryTests.m

```diff
@@ -57,8 +57,9 @@ - (void)testRegisteringFactoryAndCreatingAction
     HUBActionMock * const action = [[HUBActionMock alloc] initWithBlock:nil];
     HUBActionFactoryMock * const factory = [[HUBActionFactoryMock alloc] initWithActions:@{actionIdentifier.namePart: action}];
     [self.actionRegistry registerActionFactory:factory forNamespace:actionIdentifier.namespacePart];
-    
-    XCTAssertEqual([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
+
+    // These should be the same instance.
+    XCTAssertTrue([self.actionRegistry createCustomActionForIdentifier:actionIdentifier] == action);
```

Comment:
I think the comment is the most useful thing here. I'll revert the assert and keep the comment.